erpnext/accounts/doctype/budget/budget.js

```diff
@@ -6,7 +6,8 @@ frappe.ui.form.on('Budget', {
 		frm.set_query("cost_center", function() {
 			return {
 				filters: {
-					company: frm.doc.company
```

Comment:
hmm think budget should allow group nodes so u can allocate budgets for a range of things